nemo/collections/asr/models/transformer_bpe_models.py

```diff
@@ -367,9 +367,7 @@ def _setup_dataloader_from_config(self, config: Optional[Dict]):
                 config,
                 global_rank=self.global_rank,
                 world_size=self.world_size,
-                dataset=LhotseSpeechToTextBpeDataset(
-                    tokenizer=self.tokenizer, token_sequence_format=config.get("token_sequence_format", None),
-                ),
+                dataset=CanaryDataset(tokenizer=self.tokenizer),
```

Comment:
This will force dataset to `CanaryDataset` but sometimes users might want simply `LhotseSpeechToTextBpeDataset` instead. How about create a `Canary` model class which inherits  the `EncDecTransfModelBPE` class? Given that Canary is doing so well, I think it deserves its own model class, which will make it easier for use to update later without worrying about changing the behaviors of existing `EncDecTransfModelBPE` models. What do you think?

Reply:
Done